processor/transformer/transformer.go

```diff
@@ -409,7 +413,10 @@ func (trans *HandleT) doPost(ctx context.Context, rawJSON []byte, url string, ta
 			trans.logger.Warnf("JS HTTP connection error: URL: %v Error: %+v after %v tries", url, err, retryCount)
 		})
 	if err != nil {
-		if config.GetBool("Processor.Transformer.failOnError", false) {
+		if netErr, ok := err.(net.Error); ok && netErr.Timeout() && config.GetBool("Processor.Transformer.abortOnTimeout", false) {
```

Comment:
We only want this behaviour to happen during user transformations. Timed-out dest transformations should never result in jobs getting aborted